projects/storefrontapp-e2e-cypress/cypress/integration/b2b/regression/quick-order/b2b-quick-order.e2e-spec.ts

```diff
@@ -0,0 +1,91 @@
+import * as quickOrder from '../../../../helpers/b2b/b2b-quick-order';
+import { viewportContext } from '../../../../helpers/viewport-context';
+import * as sampleData from '../../../../sample-data/b2b-checkout';
+
+context('B2B - Quick Order', () => {
+  viewportContext(['mobile', 'desktop'], () => {
+    before(() => {
+      cy.window().then((win) => win.sessionStorage.clear());
+      cy.window().then((win) => win.localStorage.clear());
+      cy.clearLocalStorageMemory();
+    });
+
+    describe('Quick Order Page', () => {
+      beforeEach(() => {
+        quickOrder.visitQuickOrderPage();
+      });
+
+      it('should add product to the list', () => {
+        quickOrder.addProductToTheList(sampleData.b2bProduct.code);
+        quickOrder.verifyQuickOrderListQuantity(1);
+        quickOrder.clearList();
+      });
+
+      it('should add 2 different products to the list', () => {
+        quickOrder.addProductToTheList(sampleData.b2bProduct.code);
```

Comment:
in that same method, you can wait for this request to happen when adding a product to your quick order list instead of that hard-coded wait for 1s 
https://github.com/SAP/spartacus/pull/13083#discussion_r669258584

![Screen Shot 2021-07-16 at 12 33 37 AM](https://user-images.githubusercontent.com/29099466/125892379-55c8b91b-04cc-4844-b873-6ca3d251d4fd.png)

Reply:
Once you add this, it will fix the flakiness, where when it clears, they won't have a product coming out of nowhere to fail the test.

Reply:
Question at hand again is about why you need to keep the `cy.wait(1000)`

But the screenshot above explains that when you add a product to that quick order form, you are loading that exact product to your list which fires that request.

Specifically, if you look again at the screenshot below, when the test enters the product code + enter action, we can see the xhr request being made. What I'm asking is every time you 'add' a product to that list, meaning that you load it, you wait for that request to finish happening.

Example:

```
export function addProductToTheList(productCode: string) {
  #### registerIntecerpt() ---> the `GET` request which is basically 'loading' the product so that you can populate your entries

  cy.get('.quick-order-form-input input').type(`${productCode}{enter}`);

##### cy.wait(thatInterceptAlias).its('response.statusCode').should('eq', 200);
}

```

![Screen Shot 2021-07-20 at 11 30 46 PM](https://user-images.githubusercontent.com/29099466/126426416-4eb8bcf1-60d0-4039-9d5e-e0e2b6ea16c6.png)